pkg/ccl/cliccl/load_test.go

```diff
@@ -102,3 +109,54 @@ Tables:
 		}
 	})
 }
+
+func TestLoadShowIncremental(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	c := cli.NewCLITest(cli.TestCLIParams{T: t, NoServer: true})
+	defer c.Cleanup()
+
+	ctx := context.Background()
+	dir, cleanFn := testutils.TempDir(t)
+	defer cleanFn()
+	srv, db, _ := serverutils.StartServer(t, base.TestServerArgs{ExternalIODir: dir, Insecure: true})
+	defer srv.Stopper().Stop(ctx)
+
+	sqlDB := sqlutils.MakeSQLRunner(db)
+	sqlDB.Exec(t, `CREATE DATABASE testDB`)
+	sqlDB.Exec(t, `USE testDB`)
+	const backupPath = "nodelocal://0/fooFolder"
+	initTS := hlc.Timestamp{WallTime: timeutil.Now().UnixNano()}
+	initAOST := initTS.AsOfSystemTime()
+	sqlDB.Exec(t, fmt.Sprintf(`BACKUP DATABASE testDB TO $1 AS OF SYSTEM TIME '%s'`, initAOST), backupPath)
+
+	// We do two more incremental backups on the full backups.
+	incTS := hlc.Timestamp{WallTime: timeutil.Now().UnixNano()}
+	incAOST := incTS.AsOfSystemTime()
+	sqlDB.Exec(t, fmt.Sprintf(`BACKUP DATABASE testDB TO $1 AS OF SYSTEM TIME '%s'`, incAOST), backupPath)
+	incTS2 := hlc.Timestamp{WallTime: timeutil.Now().UnixNano()}
+	incAOST2 := incTS2.AsOfSystemTime()
+	sqlDB.Exec(t, fmt.Sprintf(`BACKUP DATABASE testDB TO $1 AS OF SYSTEM TIME '%s'`, incAOST2), backupPath)
+
+	out, err := c.RunWithCapture(fmt.Sprintf("load show incremental %s --external-io-dir=%s", backupPath, dir))
+	require.NoError(t, err)
+	expectedIncFolder := incTS.GoTime().Format(backupccl.DateBasedIncFolderName)
+	expectedIncFolder2 := incTS2.GoTime().Format(backupccl.DateBasedIncFolderName)
+
+	var buf bytes.Buffer
+	w := tabwriter.NewWriter(&buf, 28, 1, 2, ' ', 0)
```

Comment:
nit: Can we add some comments here like `/* minWidth */` and `/* tabWidth */` for these arguments?